MonoGame.Framework/Graphics/Vertices/VertexBuffer.cs

```diff
@@ -54,13 +54,18 @@ internal protected override void GraphicsDeviceResetting()
 
         public void GetData<T> (int offsetInBytes, T[] data, int startIndex, int elementCount, int vertexStride) where T : struct
         {
+            var elementSizeInBytes = Marshal.SizeOf(typeof(T));
+
+            if (vertexStride == 0)
+                vertexStride = elementSizeInBytes;

```

Comment:
> Talking of which, what is MonoGame's policy on doc comments? 

The same as code... we cannot use someone else's copyrighted materials.  So you cannot cut and paste over XNA docs.  And really we don't want the XNA docs... a lot of them are quite poor really.  Leaving tons of questions... we should strive for better.